src/ParseObject.js

```diff
@@ -1148,6 +1150,58 @@ class ParseObject {
     );
   }
 
+  /**
+   * Stores the object and every object it points to in the local datastore,
+   * recursively, using a default pin name: _default.
+   */
+  pin() {
+    const localDatastore = CoreManager.getLocalDatastore();
+    ParseObject.pinAllWithName(localDatastore.DEFAULT_PIN, [this]);
```

Comment:
the DEFAULT_PIN should be static somehow, it's a constant, so imported, not a property on a particular instance.